config.ru

```diff
@@ -20,16 +20,24 @@ end
 
 if log_path
   enable :logging
-  log = File.new(log_path, "a")
-  log.sync = true
-  STDOUT.reopen(log)
-  STDERR.reopen(log)
+  unless ENV["LOG_TO_STDOUT"].present?
+    log = File.new(log_path, "a")
+    log.sync = true
+    STDOUT.reopen(log)
+    STDERR.reopen(log)
+  end
 end
 
 unless in_development
-  use Rack::Logstasher::Logger,
-      Logger.new("log/production.json.log"),
-      extra_request_headers: { "GOVUK-Request-Id" => "govuk_request_id", "x-varnish" => "varnish_id" }
+  unless ENV["LOG_TO_STDOUT"].present?
+    use Rack::Logstasher::Logger,
+        Logger.new("log/production.json.log"),
+        extra_request_headers: { "GOVUK-Request-Id" => "govuk_request_id", "x-varnish" => "varnish_id" }
+  else
+    use Rack::Logstasher::Logger,
+        Logger.new($stdout),
+        extra_request_headers: { "GOVUK-Request-Id" => "govuk_request_id", "x-varnish" => "varnish_id" }
+  end
```

Comment:
Here perhaps you can do

```ruby
logger = ENV.key?("LOG_TO_STDOUT") ? Logger.new($stdout) : Logger.new("log/production.json.log")
use Rack::Logstasher::Logger,
        logger,
        extra_request_headers: { "GOVUK-Request-Id" => "govuk_request_id", "x-varnish" => "varnish_id" }
```

to avoid duplication

Reply:
looks good